src/main/java/com/bigchaindb/builders/BigchainDbTransactionBuilder.java

```diff
@@ -88,6 +88,8 @@ public static Builder init() {
 
         ITransactionAttributes addInput(String fullfillment, FulFill fullFill, EdDSAPublicKey publicKey);
 
+        ITransactionAttributes addInput(Details fullfillment, FulFill fullFill, EdDSAPublicKey... publicKey);
```

Comment:
adds interface support for `addInput(Details ...)`, not just with a `String` fulfillment. the intent here is to make it easier for the user to construct an `Input` from a previous transaction, when they already have `Details` laying around.

Reply:
accepted